### DISCUSSION TITLE
Time based plays

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#5768] [Unsupported Versions] [answered]
Hello!
Question is: is there any way to make time based plays in playbook? For example, I want to detect logon in some systems in non-working time.
The only time-related field in sigma is `timeframe`, but it's not working as I see and doesn't cover my search. Second way is use some script that will activate such rules after work and disable it before work, but it's really strange. So is there any more convenient way? Thanks!

UPD: same question for multiple events in period of time: any way to alert about 10 logons within 5 minutes, for example?
UPD 2: found more sigma fields for aggregation, but is seems that they don't work in ELK/SO.

**Comment by Sahale** [comment, marked as answer]
So, after creation of such rule, I've edited ElastAlert rule via playbook, but it didn't help. So I found yaml file, changed type to frequency and added `timeframe` and `num_events` fields. Now it works.
Maybe it should be described somewhere in wiki?

**Reply by tiger-greenhell** [reply]
I agree, it might be useful to others.

**Reply by dougburks** [reply]
@Sahale Please feel free to submit pull requests for the documentation:
https://github.com/Security-Onion-Solutions/securityonion-docs

Or simply post your recommended changes here and we will add them to the documentation for you.
